torch/distributed/algorithms/ddp_comm_hooks/default_hooks.py

```diff
@@ -65,3 +66,33 @@ def decompress(fut):
         return [decompressed_tensor]
 
     return fut.then(decompress)
+
+def fp16_compress_wrapper(hook: Callable[..., torch.futures.Future]) -> Callable[..., torch.futures.Future]:
```

Comment:
Can you use `Callable[[Any, dist.GradBucket], torch.futures.Future]` as the hook type instead?

Reply:
Yep that's better